docs/data-warehouse-api/resources/tracker/raw_data.md

```diff
@@ -71,90 +84,127 @@ curl -X 'POST' \
+| server_time_filter | string/object | An interval for additional filtering of messages by server time. If used, messages will be returned not only based on the message time (when the packet was registered by a tracker), but also filtered by the server time (when the message was received by the server).                                              | `"2024-02-03T10:26:26+0500/2024-02-03T10:27:18+0500"` / `{"from": "2024-02-03T10:26:26+0500", "to": "2024-02-03T10:27:18+0500"}` / `{"interval":"2024-02-03T10:26:26+0500/PT1H"}` | no       |
+| format             | string        | This is used when you need to request data in `parquet` format. If the format parameter is not provided, the default format used will be CSV.                                                                                                                                                                          | `"parquet"`                                                                                                                                                                       | no       |
 
-!!! note "Instead of using `from`/`to` parameters it is possible to set `interval` parameter - ISO 8601 formatted interval, for example 2023-08-24T08:04:36.306Z/PT24H."
+!!! note "Instead of using the `from`/`to` parameters, you can set the `interval` parameter — an ISO 8601 formatted interval, for example, `2023-08-24T08:04:36.306Z/PT24H`."
+
+The response could be provided in a CSV or [Parquet](https://parquet.apache.org/docs/overview/) format file, with columns 
+predefined in the `columns` parameter of the API request. 
 
-The response is provided in a CSV format file, with columns that are predefined in the `columns` parameter of the API request.
 Here are the specifications for the table output:
 
 * Rows are enclosed in double quotes. A double quote inside a string is output as two double quotes in a row. There are no other rules for escaping characters.
-* Date and date-time are enclosed in double quotes.
+* Date and date-time values are enclosed in double quotes.
 * Numbers are output without quotes.
 * Values are separated by a comma character `,`.
 * Rows are separated using the Unix line feed (LF).
 * `NULL` is represented as `\N`.
 * Requested column can be a simple or complex.
 
-**Simple** columns:
-
-* `msg_time` - time of message sent by the device. Always returned in CSV output and does not need to be requested separately. Indicated according to user account time zone.
-* `server_time` - time of message processing by the server.
-* `gps_fix_type` - enum. One of `UNKNOWN`, `NO_FIX`, `HAS_FIX`, `LAST_KNOWN_POSITION`.
```

Comment:
@Astarsys also, in the parquet format enums represented as integers

gps_fix_type - UNKNOWN=0, NO_FIX=1, HAS_FIX=2, LAST_KNOWN_POSITION=3
mn_type - UNKNOWN=0, GSM=1, CDMA=2, WCDMA=3, LTE=4, NR=5)

Reply:
sure, I added it under the table.

Reply:
oops, sorry